src/main/riddl/Venues/Reservations/reservationMessages.riddl

```diff
@@ -0,0 +1,45 @@
+command EstablishReservation is {
```

Comment:
Maybe MakeReservation? I think that is how our users would call this action...

Reply:
If we are going that realistic I think BookReservation would be appriopriate